homeassistant/components/devolo_home_control/strings.json

```diff
@@ -5,18 +5,18 @@
       "already_configured": "This Home Control Central Unit is already in use."
     },
     "error": {
-      "invalid_credentials": "Incorrect user name and/or password."
+      "invalid_credentials": "[%key:common::config_flow::error::invalid_auth%]"
     },
     "step": {
       "user": {
         "data": {
-          "username": "E-Mail-Address / devolo ID",
+          "username": "[%key:common::config_flow::data::email%] / devolo ID",
           "password": "[%key:common::config_flow::data::password%]",
-          "mydevolo_url": "mydevolo URL",
-          "home_control_url": "Home Control URL"
+          "mydevolo_url": "mydevolo [%key:common::config_flow::data::url%]",
```

Comment:
Not sure this is really necessary

Reply:
Shall I keep the old version? Obviously we need to distinguish two URLs. I'm fine with either way.

Reply:
Sure we need to distinguish both URLs. But i'm not sure it's allowed to get a partial translation with reference strings.

Reply:
It's allowed/supported